core/io/image.cpp

```diff
@@ -4064,8 +4077,8 @@ Dictionary Image::compute_image_metrics(const Ref<Image> p_compared_image, bool
 
 	ERR_FAIL_COND_V(err != OK, result);
 
-	ERR_FAIL_COND_V_MSG((compared_image->get_format() >= Image::FORMAT_RH) && (compared_image->get_format() <= Image::FORMAT_RGBE9995), result, "Metrics on HDR images are not supported.");
-	ERR_FAIL_COND_V_MSG((source_image->get_format() >= Image::FORMAT_RH) && (source_image->get_format() <= Image::FORMAT_RGBE9995), result, "Metrics on HDR images are not supported.");
+	ERR_FAIL_COND_V_MSG(compared_image->is_hdr(), result, "Metrics on HDR images are not supported.");
+	ERR_FAIL_COND_V_MSG(source_image->is_hdr(), result, "Metrics on HDR images are not supported.");
```

Comment:
Was the previous check outdated? Otherwise it changes behavior with this change, or is it just a subrange of the formats already checked?

Reply:
To be honest, I'm not sure whether this should be changed or left as-is (along with the other instances of float being treated like ldr), since that depends on whether the original intent was to handle float like a normalized format within the 0-1 range, and not like half which looks like it was meant to be a purely hdr format.

Reply:
I'd say to leave it as is and evaluate that as a separate thing, safest that way